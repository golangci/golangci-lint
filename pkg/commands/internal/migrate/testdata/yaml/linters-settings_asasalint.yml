issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  asasalint:
    exclude:
      - Append
      - \.Wrapf
    use-builtin-exclusions: false
    ignore-test: true
