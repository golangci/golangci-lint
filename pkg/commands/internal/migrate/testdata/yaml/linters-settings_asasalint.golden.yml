version: "2"
linters:
  settings:
    asasalint:
      exclude:
        - Append
        - \.Wrapf
      use-builtin-exclusions: false
  exclusions:
    rules:
      - linters:
          - asasalint
        path: (.+)_test\.go
